src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -469,8 +470,8 @@ public void verifyUserLoggedIn(final String userName) {
         driver.switchTo().frame("PreviewFrame");
       }
       // open nav if on mercury, required to see login data
-      if (driver.findElements(MERCURY_NAV_ICON).size() > 0) {
-        wait.forElementClickable(MERCURY_NAV_ICON);
+      if (driver.findElements(MERCURY_SKIN).size() > 0) {
```

Comment:
Yes, here I would just like to check whether it's Mercury or not. I don't want to wait for anything here. I want to wait in the line below